packages/web-components/src/calendar/calendar.spec.md

```diff
@@ -0,0 +1,126 @@
+| `work-week`                | `[0, 1, 2, 3, 4, 5, 6]`                                 | `[1, 2, 3, 4, 5]` | The days that are selectable with the filter `work-week`. If the filter is not set to `work-week`, this does nothing.             |
+| `first-day-of-week`        | `0, 1, 2, 3, 4, 5, 6`                                   | `0`               | The first day of the week for locale                                                                                              |
+| `first-week-of-year`       | `0, 1, 2`                                               | `0`               | Enum that defines when the first week of the yera should start: first day (`0`), first full week (`1`), first four day week (`2`) |
+| `min-date`                 | string                                                  | -                 | The minimum allowable date                                                                                                        |
+| `max-date`                 | string                                                  | -                 | The maximum allowable date                                                                                                        |
+| `show-link`                | boolean                                                 | `true`            | If false, link at link slot is hidden                                                                                             |
+| `highlight-current-month`  | boolean                                                 | -                 | if true, the month picker should highlight the current month                                                                      |
+| `highlight-selected-month` | boolean                                                 | -                 | If true, the month picker should highlight the selected month                                                                     |
+| `today`                    | string                                                  | -                 | value of today                                                                                                                    |
+| `selected-dates`           | string[]                                                | -                 | selected dates                                                                                                                    |
+
+### Outputs
+
+- [selectedDates: string] - the selected dates from the calendar
+
+### Events
+
+- `change`: Emits when the selected date(s) is changed
+- `date-select`: Emits when user selects a date(s) when no date(s) is currently selected
+- `view-change`: Emits when view changes (e.g. from date picker to month picker) via user interaction
+
+### Slots
+
+- `start` - content before the calendar
```

Comment:
Do we need start and end with the calendar component? I think in the case of a calendar this could be handled through composition. Or, do these start and end slots embed content inside the calendar?

Reply:
The start and end slots don't embed content inside the calendar. I may have misunderstood start and end slots, but I thought that they should be provided for anything that comes before or after the component? In this case, there are no prescriptive content that should go before or after the calendar. Would this mean that there would be no start or end slots?

Reply:
> Do we need start and end with the calendar component? I think in the case of a calendar this could be handled through composition. Or, do these start and end slots embed content inside the calendar?

The start and end slots don't embed content inside the calendar. I may have misunderstood start and end slots, but I thought that they should be provided for anything that comes before or after the component? In this case, there are no prescriptive content that should go before or after the calendar. Would this mean that there would be no start or end slots?

Reply:
Start and end slots are typically for embedding inside the component. A classic use case is a button where the dev wants to insert an icon inside the button. The web component template would be something like this:

```
<button>
    <slot name="start"></slot>
    <slot></slot>
    <slot name="end"></slot>
</buttom>
```

If the slots are out of the calendar bounds then the dev can handle layout through composition and doesn't need slots:

```
<div class="my-calendar">
    <some-component-start></some-component-start>
    <calender></calendar>
    <some-component-end></some-component-end>
<div>
```

Reply:
Ah I see, that makes sense! Appreciate the example!